data/hai/hai ships.txt

```diff
@@ -733,6 +789,57 @@ ship "Pond Strider" "Pond Strider (Railgun)"
 
 
 
+ship "Scarab"
+	sprite "ship/hai scarab"
+	thumbnail "thumbnail/hai scarab"
+	attributes
+		category "Transport"
+		"cost" 1720000
+		"shields" 4500
+		"hull" 1600
+		"required crew" 5
+		"bunks" 27
+		"mass" 146
+		"drag" 3.8
+		"heat dissipation" .87
+		"fuel capacity" 500
```

Comment:
```suggestion
		"cost" 1620000
		"shields" 4900
		"hull" 1000
		"required crew" 4
		"bunks" 27
		"mass" 146
		"drag" 3.8
		"heat dissipation" .87
		"fuel capacity" 400
```
For the Scarab, also drop the cost. 
Make the same change to shields/hull, also 200 less EHP
Drop the required crew by one since it's a pretty small ship
Drop some fuel